app/javascript/spec/miq-structured-list/__snapshots__/miq-structured-list.spec.js.snap

```diff
@@ -1972,3 +1972,52 @@ exports[`Structured list component should render a tag_group table 1`] = `
   </AccordionItem>
 </Accordion>
 `;
+
+exports[`Structured list component should render strings without double exiting strings 1`] = `
+<FeatureToggle(StructuredListWrapper)
+  ariaLabel="Structured list"
+  className="miq-structured-list miq_summary miq_host_devices"
+>
+  <FeatureToggle(StructuredListBody)>
+    <FeatureToggle(StructuredListRow)
+      className=""
+      key="0"
+      onClick={[Function]}
+      tabIndex={0}
+      title=""
+    >
+      <FeatureToggle(StructuredListCell)
+        className="label_header"
+      >
+        Catch All
+      </FeatureToggle(StructuredListCell)>
+      <FeatureToggle(StructuredListCell)
+        className="content_value object_item"
+      >
+        <div
+          className="content"
+        >
+          <div
+            className="cell icon"
+          >
+            <i
+              className="pficon pficon-unknown"
+              style={
+                Object {
+                  "background": undefined,
+                  "color": undefined,
+                }
+              }
+            />
+          </div>
+          <div
+            className="expand wrap_text"
+          >
+            &lt; A test to HTML's "Escape" logic Hello&Goodbye &gt;
```

Comment:
How weird - I expected the &amp; here 😕

Reply:
Me too because its clearly escaping the `< >` but not the `&` for some reason

Reply:
(though from what i saw online it seems its not escaping any of those symbols https://www.educative.io/answers/how-to-escape-unescape-html-characters-in-string-in-javascript ? )

Reply:
As discussed it seems react is smart enough to know when it _must_ escape and when it's optional, as opposed to Rails' `h` method which always does it.